test/unit/modules/signatures.js

```diff
@@ -14,66 +14,226 @@
 
 'use strict';
 
+const application = require('../../common/application');
+const transactionTypes = require('../../../helpers/transaction_types');
+const ApiError = require('../../../helpers/api_error');
+
 /* eslint-disable mocha/no-pending-tests */
+/* eslint-disable no-unused-vars */
```

Comment:
Why is it important to keep?

Reply:
I added it due to ESLint being picky when not using the last arguments on callback definitions. For example [here](https://github.com/LiskHQ/lisk/blob/00735eb98e22e4b4d8b007036f30d150f6750fcd/test/unit/modules/signatures.js#L160) and [here](https://github.com/LiskHQ/lisk/blob/00735eb98e22e4b4d8b007036f30d150f6750fcd/test/unit/modules/signatures.js#L195). Also it wouldn't let me type "_" as the name of the arguments to dismiss those errors.

Reply:
If possible, please do not disable rules on file level. Disable the relevant lines instead with eslint-disable-next-line

Reply:
Fixed